visualization/eolearn/visualization/eoexecutor_visualization.py

```diff
@@ -36,14 +36,13 @@ class EOExecutorVisualization:
     """ Class handling EOExecutor visualizations, particularly creating reports
     """
 
-    def __init__(self, eoexecutor):
+    def __init__(self, eoexecutor: 'eolearn.core.EOExecutor'):
```

Comment:
My linters complain about this annotation. Here you can just import the class and annotate with it, so that linters can verify the rest of the code

Reply:
Fixed